### ISSUE TITLE
Cannot provide ITextFormatter directly

### ISSUE THREAD

**Issue by tom-val** [manigandham/serilog-sinks-fastconsole#5]
With current project implementation, I can only provide different template using "outputTemplate". But for my project I need to provide ElasticsearchJsonFormatter directly.
It looks like this for regular Console output:
`.WriteTo.Console(formatter: new ElasticsearchJsonFormatter(inlineFields: true, formatStackTraceAsArray: false))`
It would be nice to be able to provide formatter directly, since "outputTemplate" does not really work for me

**Comment by manigandham** [comment]
You can, `ITextFormatter` is already supported as a parameter: 
https://github.com/manigandham/serilog-sinks-fastconsole/blob/master/src/Serilog.Sinks.FastConsole/FastConsoleSink.cs#L23

The parameter is called `textFormatter` in this sink. You can pass in the config like this:

```csharp
var config = new FastConsoleSinkOptions();
.WriteTo.FastConsole(config, new ElasticsearchJsonFormatter(inlineFields: true, formatStackTraceAsArray: false));
```

**Comment by tom-val** [comment]
Thanks for fast response!
It's added to sink itself, but it's not added in extension:
https://github.com/manigandham/serilog-sinks-fastconsole/blob/master/src/Serilog.Sinks.FastConsole/FastConsoleSinkExtensions.cs
Unless I'm missing something here?

**Comment by manigandham** [comment]
Ah yea you're right, I never fixed this. I'll get it updated.

**Comment by manigandham** [comment]
@tom-val 
This has been released as 2.0.0: https://www.nuget.org/packages/Serilog.Sinks.FastConsole/2.0.0

**Comment by tom-val** [comment]
Thanks, works as expected now
